SD_Assignment1/src/main/java/ro/utcn/aron/model/Question.java

```diff
@@ -0,0 +1,47 @@
+package ro.utcn.aron.model;
+
+public class Question {
+
+	private int id;
+	private String title;
+	private String body;
+	private String tags;
+	
+	
+	public Question(int id, String title, String body, String tags) {
+		this.id = id;
+		this.title = title;
+		this.body = body;
+		this.tags = tags;
+	}
+	
+
+	public int getId() {

```

Comment:
[Optional] Use Lombok for getter and setter, constructors, etc. It is not an excuse that you do not want to install it in your IDE; it is used in most Java projects and you will have to install the toolchain when you will work on a real project.